src/packaging/resource/cassandra-reaper-cassandra-ssl.yaml

```diff
@@ -64,6 +65,14 @@ cassandra:
   clusterName: "test"
   contactPoints: ["127.0.0.1"]
   keyspace: reaper_db
+  loadBalancingPolicy:
+    type: tokenAware
+    shuffleReplicas: true
+    subPolicy:
+      type: dcAwareRoundRobin
+      localDC:
+      usedHostsPerRemoteDC: 0
+      allowRemoteDCsForLocalConsistencyLevel: false
```

Comment:
i thought these were already the default settings in the code?

if not, then yes to the change,

but if so, is there a reason to explicit repeatly the default settings into the config? 
(this would make it harder to change it in the code and to be applied in new version updates)